app/components/ProjectEntryPoint/ProjectEntryPoint.js

```diff
@@ -35,10 +35,13 @@ const projectEntryPoint = (props) => {
     findEntryPoints(assets, entryPointsList);
   }
 
+  const sep = process.platform === 'win32' ? '\\' : '/';
```

Comment:
I think you can use the built-in `path.sep` instead of deriving it yourself (https://nodejs.org/docs/latest/api/path.html#pathsep)

Reply:
Ohk, I guess this would surely work. Thanks